evergreen/run-mongodb-aws-test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit  # Exit the script with error if any of the commands fail
+
+############################################
+#            Main Program                  #
+############################################
+
+echo "Running MONGODB-AWS authentication tests"
+
+# Provision the correct connection string and set up SSL if needed
+for var in TMP TEMP NUGET_PACKAGES NUGET_HTTP_CACHE_PATH APPDATA; do setx $var z:\\data\\tmp; export $var=z:\\data\\tmp; done
+
+# ensure no secrets are printed in log files
+set +x
+
+# load the script
+shopt -s expand_aliases # needed for `urlencode` alias
+[ -s "${PROJECT_DIRECTORY}/prepare_mongodb_aws.sh" ] && source "${PROJECT_DIRECTORY}/prepare_mongodb_aws.sh"
+
+MONGODB_URI=${MONGODB_URI:-"mongodb://localhost"}
```

Comment:
Can we have empty MONGODB_URI at all? And even if so, should we update the up level script to fill it? As I understand, we don't fill connection string on this level usually

Reply:
if we don't want to miss a bug in the our configuration when we missed filling of MONGODB_URI, maybe it's better to trigger exception in this case?